junit5/src/main/java/com/linecorp/armeria/testing/junit5/server/ServerExtension.java

```diff
@@ -29,12 +29,15 @@
 import com.linecorp.armeria.internal.testing.ServerRuleDelegate;
 import com.linecorp.armeria.server.Server;
 import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.ServiceRequestContext;
 import com.linecorp.armeria.testing.junit5.common.AbstractAllOrEachExtension;
 
 /**
  * An {@link Extension} that allows easy set-up and tear-down of a {@link Server}.
  */
 public abstract class ServerExtension extends AbstractAllOrEachExtension {
```

Comment:
Could we provide a way to disable the capture? For example, we could have a method like this:

```java
protected boolean shouldCapture(ServiceRequestContext ctx) {
    return true;
}
```